fleetspeak/src/server/components/proto/fleetspeak_components/config.proto

```diff
@@ -80,11 +80,9 @@ message AdminConfig {
 }
 
 message StatsConfig {
-  // Stats collector to be used. Optional, but recommended.
-  // Defaults to no stats collector.
-  string collector = 1;
-
-  // The bind address to listen on for connections, e.g. ":443" or
-  // "localhost:1234". Optional, defaults to ":2112".
-  string port = 2;
+  // The bind address to listen on for connections in the
```

Comment:
nit: suggest mentioning the type of connection, e.g. "to listen on for Prometheus http metric collection"

Reply:
Fixed in 1d8be41.